model/result.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from "zod";
+
+export enum Result {
+  WIN = "You Win!!!",
+  LOSE = "You lose :< ",
+  DRAW = "It's a Draw!",
+}
+
+const ResultEnum = z.nativeEnum(Result);
+export type ResultEnum = z.infer<typeof ResultEnum>;
```

Comment:
Generally speaking: same as above.

Another gotcha here: since you're not using any runtime feature for `Result` (you're not parsing the result anywhere), you probably don't need zod at all.